src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -371,11 +371,22 @@ void drawing_classifier::init_training(
   // by the data iterator.
   nn_spec_ = init_model();
 
-  if ( enable_warmstart ) {
+  if (mlmodel_path.size()) {
     // Initialize the neural net with warm start model weights.
     model_spec warmstart_model(mlmodel_path);
     float_array_map trained_weights = warmstart_model.export_params_view();
     nn_spec_->update_params(trained_weights);
+
+    size_t idx = 0;
+#ifdef _WIN32
```

Comment:
not sure we support windows or not. @nickjong @shantanuchhabra

Reply:
Let's just avoid doing path manipulations here